ktx-mapbox-maps/gradle.properties

```diff
@@ -2,4 +2,4 @@ VERSION_NAME=0.1.0-SNAPSHOT
 POM_ARTIFACT_ID=mapbox-android-sdk-ktx-v7
 POM_NAME=Mapbox Maps SDK for Android KTX
 POM_DESCRIPTION=A set of Kotlin extensions for the Mapbox Maps SDK for Android.
-POM_PACKAGING=aar
\ No newline at end of file
+REGISTRY_SDK_NAME=mapbox-android-ktx
```

Comment:
is this the correct name? looks different than the other names.

Reply:
It seems we have never released this before. Please correct me if I'm wrong. @tobrun